src/eduid_userdb/actions/tou/user.py

```diff
@@ -46,7 +46,13 @@ def __init__(self, userid = None, tou = None, data = None):
         if data is None:
             data = dict(_id = userid,
                         tou = tou,
+                        #  We do not have a eppn here, but it is required for User.
+                        #  Since from the tou db we only update the central db with
+                        #  ToUs, this dummy eppn has no effect.
```

Comment:
Yes it is ugly, should be harmless but very ugly. I'll make it so that we do not have to call super __init__() from ToUUser.__init__